examples/appdir/src/app/page.tsx

```diff
@@ -45,6 +45,9 @@ export default function Home() {
           //   },
           // }}
           className="ut-button:bg-red-400 ut-allowed-content:text-lg ut-button:ut-readying:bg-orange-300 rounded border p-3"
+          onUploadBegin={(name) => {
+            console.log("Uploading: ", name);
```

Comment:
I would maybe add "onUploadBegin:" to the log here, just so anything in the console is more clear where it is coming from.

Reply:
Okay

Reply:
@markflorkowski done

Reply:
Did your latest commit mess up formatting?

We shoudl add a ci step for that 😅

Reply:
Sorry, just did run pnpm format and pushed